core/src/main/java/org/apache/druid/java/util/common/guava/MergeSequence.java

```diff
@@ -46,22 +46,20 @@ public MergeSequence(
   @Override
   public <OutType> Yielder<OutType> toYielder(OutType initValue, YieldingAccumulator<OutType, T> accumulator)
   {
-    PriorityQueue<Yielder<T>> pQueue = new PriorityQueue<>(
-        32,
-        ordering.onResultOf(
-            new Function<Yielder<T>, T>()
-            {
-              @Override
-              public T apply(Yielder<T> input)
-              {
-                return input.get();
-              }
-            }
-        )
-    );
-
-    pQueue = baseSequences.accumulate(
-        pQueue,
+    PriorityQueue<Yielder<T>> pQueue = baseSequences.accumulate(
+        new PriorityQueue<>(
+            32,
```

Comment:
If anyone finds this comment through code archeology, this constant was me just poking around and trying stuff. It is not thoroughly tested :(

Reply:
Thanks. Good to know.